src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,111 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private int capasitu = 16;
+    private float loadFactor = 0.75f;
+    private int size;
+    private Node<K,V>[] table;
```

Comment:
The 'table' variable doesn't have an access modifier. It's a good practice to always specify an access modifier to follow the encapsulation principle. In this case, it should be 'private'.